robot/install.bash

```diff
@@ -1,4 +1,17 @@
 #! /usr/bin/env bash
+[[ "$CI" == "true" ]] && return 0
+
+if [[ "$ROBOT_REAL" != "true" ]]
+  then
+    read -r -p "This package ($target) should only be installed on the robot itself, not on your own PC. Are you sure to continue? [y/N] " response
+    response=${response,,}    # tolower
+    if [[ "$response" =~ ^(yes|y)$ ]]
+    then
+      :
+    else
+      return 1
+    fi
+fi
```

Comment:
Where is `ROBOT_REAL` set? As far as I know in robot/setup. Which is only sourced after a successful install.

Reply:
Interesting..
We actually do the check on the username so didn't have that problem. But I changed this to be more descriptive.
I can change it back to `"$USER" != "amigo"`?

Reply:
That will not work as many people have amigo as username.

Reply:
Shall we come up with a solution or don't we need this kind of warning?
I guess if we solve [that](https://github.com/tue-robotics/tue-env/pull/449#discussion_r373800957), than this is also solved.